nova/core/roles/os_configuration/defaults/main.yml

```diff
@@ -18,13 +16,8 @@ os_configuration_wait_for_site_url: https://google.com # URL to wait for
 os_configuration_wait_for_site_retries: 30 # How many times to retry waiting for site defined in os_configuration_wait_for_site_url respond with HTTP 200 before moving on with Linux hosts
 os_configuration_expand_os_disk_role_name: {} # Define your OS disk expansion role name in your host or group vars to included it with this role
 fortios_access_token: {} # Required token to configure FortiOS
-
-configure_chocolatey_sources: false # Set to true and follow the configuration example below to set Chocolatey sources
-chocolatey_sources:
-  - name: chocolatey-proxy
-    state: present
-    source: "{{ chocolatey_proxy_uri }}"
-    priority: 1
+os_configuration_post_roles: [] # List of extra roles to be included as one of the last steps of this role
```

Comment:
Is this not too hidden? there is the include extra roles functionality in finalize already

Reply:
I'll add this info also to the readme. The functionality in finalize is if for tasks that need to happen at the very end of the deploy for an example adding disk encryption. If this gets included earlier then the deploy will fail if the machine reboots and disk is encrypted. As for the `os_configuration_post_roles` since there is the capability to use different deploy modes like `just-clone` having a similar post roles feature in this role is useful because users can further customize their deployment tree for an example configuring package mirrors with a separate role before the `just-clone` deploy mode creates a snapshot of the machine.